### PR TITLE
CVSL-1879 Remove hardstop banner

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: helm_deploy/values-prod.yaml [hunk 1]
```diff
@@ -26,13 +26,10 @@ generic-service:
     GOTENBERG_API_URL: "http://create-and-vary-a-licence-gotenberg"
     # A reference to the CVL service - for links in HTML to get images/stylesheets (internal within namespace)
     LICENCES_URL: "http://create-and-vary-a-licence"
-    RESTRICT_ROLLOUT: "false"
```
Comment (andrewrlee) [inline helm_deploy/values-prod.yaml:29]: this isn't referred to in the code base

File: helm_deploy/values-dev.yaml [hunk 2]
```diff
@@ -53,7 +53,6 @@ generic-service:
 
     # To enable common front-end components
     COMPONENT_API_URL: "https://frontend-components-dev.hmpps.service.justice.gov.uk"
-    COMMON_COMPONENTS_ENABLED: "true"
```
Comment (andrewrlee) [inline helm_deploy/values-dev.yaml:56]: inheriting this from values.yml

File: helm_deploy/create-and-vary-a-licence/values.yaml [hunk 3]
```diff
@@ -52,17 +52,18 @@ generic-service:
...
     # To enable common front-end components
     COMPONENT_API_URL: 'https://frontend-components-dev.hmpps.service.justice.gov.uk'
-    COMMON_COMPONENTS_ENABLED: 'true'
     MANAGE_USERS_API_URL: 'https://manage-users-api-dev.hmpps.service.justice.gov.uk'
     FRIDAY_RELEASE_POLICY: 'https://www.gov.uk/government/publications/discretionary-fridaypre-bank-holiday-release-scheme-policy-framework'
+    # Feature toggles
+    LICENCE_WATERMARK: "false"
+    TOKEN_VERIFICATION_ENABLED: "true"
+    COMMON_COMPONENTS_ENABLED: "true"
+    SHOW_WHATS_NEW_BANNER: "false"
```
Comment (andrewrlee) [inline helm_deploy/create-and-vary-a-licence/values.yaml:65]: disabling banner

**Review by nishanthmahasamudram — APPROVED** [review]
LGTM

**Pull request merged**